components/notifications.js

```diff
@@ -0,0 +1,41 @@
+// node_modules
+import PropTypes from "prop-types";
+import {
+  DataArea,
+  DataAreaTitle,
+  DataItemLabel,
+  DataItemValue,
+  DataPanel,
+} from "./data-area";
+/**
+ * display notifictions when query is failed
+ */
+export default function Notifications({ notifications }) {
+  return (
+    <>
+      <DataAreaTitle>Error Notifications</DataAreaTitle>
+      <DataPanel>
+        <NotificationsList notifications={notifications}></NotificationsList>
+      </DataPanel>
+    </>
+  );
+}
+
+Notifications.propTypes = {
+  // notifications to display
+  notifications: PropTypes.object.isRequired,
+};
+
+function NotificationsList({ notifications }) {
+  return Object.keys(notifications).map((note, noteIdx) => (
+    <DataArea key={noteIdx}>
```

Comment:
I don’t know what form the notification object takes. If each note has some sort of guaranteed unique value, better to use that as the key than an index. If this rerenders with the notifications in a different order, this component won’t show the new order because the keys remain 0, 1, 2…. But if you can’t guarantee any property is unique to each note, then the index is all you can do.